README.md

```diff
@@ -1,2 +1,25 @@
 # minimo
-tiny mocap utilities
+
+Building the most simple and robust tools for doing typical vtuber things. 
+
+Slapping an MIT License and competing through sheer rate of iteration.
+
+# Resources
+Maintaining a list of existing work so we know what we're up against. 
+
+## writing
+- [BestVTuberSoftware gist from emilianavt](https://gist.github.com/emilianavt/cbf4d6de6f7fb01a42d4cce922795794)
+- [Runa Penumbra’s VTuber References](https://docs.google.com/spreadsheets/d/15UpI8GEqv22T45AD3L1EgcPB4l2Tvr64aDgLq7xZfMA/view#gid=0)
+- [Virtual Motion Capture Protocol](https://protocol.vmc.info/english) (VMCProcotol, OSC/VMC Protocol)
+
+## software
+- [VeeSeeFace](https://www.vseeface.icu/)
+- [OpenSeeFace](https://github.com/emilianavt/OpenSeeFace)
+- [waidayo](https://github.com/nmchan/waidayo) - not OSS and unmaintained
+- [Unity ARKit Plugin](https://github.com/U3DC/Unity-ARKit-Plugin) - is this even relevant anymore?
+
+---
+
+Author's Note:
+
+_I'm not really huge "vtuber" guy, but wouldn't it be cool to have an expressive digital persona? From an outsider's perspective, it looks like all the mainstream apps and tools add so much bloat and complexity (i.e. game engines, plugins, protocols). I could be totally wrong, but let's just build and see._
```

Comment:
I feel like physics are nice, but if I just want "expressiveness" - that just takes some creativity.